CMakeLists.txt

```diff
@@ -1,23 +1,65 @@
 cmake_minimum_required(VERSION 3.12) #Python3 package locator only added in 3.12
 
+project(cmakedoc DESCRIPTION "CMake automatic documentation generator" LANGUAGES NONE) #Not using any compiled languages so don't list, removes check for compilers
```

Comment:
you want to also stick `VERSION 1.0.0` in here so CMake can version it correctly.

Reply:
Added, but having two version numbers (the other is in setup.py) will be cumbersome. I'll put the version in a separate `version.txt` file and then pull from that in both files

Reply:
Version is now stored in `version.txt`, both CMake and setup.py pull from it.